packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -440,7 +441,17 @@ export default class TabAdvanced extends Component {
                         value={migrationNetwork}
                         xoType='network'
                       >
-                        {migrationNetwork !== undefined ? <Network id={migrationNetwork.id} /> : _('noValue')}
+                        {pool.otherConfig['xo:migrationNetwork'] === undefined ? (
+                          _('noValue')
+                        ) : migrationNetwork !== undefined ? (
+                          <Network id={migrationNetwork.id} />
+                        ) : (
+                          <span className='text-danger'>
+                            {_('networkIdMissing', {
+                              networkID: <Copiable>{pool.otherConfig['xo:migrationNetwork']}</Copiable>,
+                            })}
+                          </span>
+                        )}
```

Comment:
When you click on the copy button, it open the selector to update the network. Is that possible to fix that behavior?
![Peek 24-01-2024 11-04](https://github.com/vatesfr/xen-orchestra/assets/70369997/138c7065-4c28-4d4e-b2ba-a1dd95bbbd72)